### PR TITLE
Webpack: Cleanup common config

### PR DESCRIPTION
**Pull request by davidmehren**
Just a small cleanup of the common webpack config.

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
+const webpack = require('webpack')
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
-// Fix possible nofile-issues
-var fs = require('fs')
-var gracefulFs = require('graceful-fs')
-gracefulFs.gracefulify(fs)
```
Comment (SISheogorath) [inline webpack.common.js]: I would like to keep this.

image

Comment (davidmehren) [reply webpack.common.js]: Ahh, that's what this is for ^^

Let's keep removing this out of this PR then. Do we have an open Issue to get to the bottom of the error?

Comment (SISheogorath) [reply webpack.common.js]: I don't think so.

https://github.com/systemd/systemd/pull/10244 might solves it soon, without touching any (Node)JS code :smile: 

Apart from that, I guess it's simply related to the size of our codebase :/

**Comment by SISheogorath** [comment]
I like it :)

**Pull request merged**